Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
A prototype is only an instance of a class that you will want to copy so that it preserves the default values and then change the rest. I see no problem in changing it at run time, when we need to change a default for all the new instances created from a prototype or to add new prototypes that we will later copy, or even remove ones that we don't want anymore.

The issue here is that technically we don't need to clone the prototypes, since at the moment they don't hold data that will change at some point. But that will change when we expand the system.

The other issue, is that this function did 2 totally different things depending on a parameter. It should either always add the Headline to prototypes and show it now, or never add it and show it now or show it at a later time. Or have 2 functions.